index.html

```diff
@@ -0,0 +1,194 @@
+            A joyful celebration bringing kids together to celebrate togetherness and forget everything else,<br>A child has a special way of adding joy to everyone. The kids who want the most love, will ask for it in the most unloving way. Sometimes we have to go deep inside ourselves to solve our problems. And so a live concert is exciting because of all the electricity that is generated in the crowd and on stage. It is the favourite part of the live concerts.
+          </p>
+        </div>
+        <div class="event-details">
+          <h3>2021.09.11(SAT) ~ 12(SUN)</h3>
+          <p class="venue font-lato">
+            @ National Theatre, Kampala, Uganda.
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="second-section">
+    <div class="second-section-container">
+      <h2 class="second-section-heading">Concert Program</h2>
+      <hr />
+      <div class="program-list">
+        <div class="lecture flex-box">
+          <img class="flex-box-img icons" src="./images/game1.png" alt="" />
+          <h2>Bike Riding</h2>
+          <p>
+            The best ages to teach your child are between four and six years old. It should take 45 minutes.
+          </p>
+        </div>
+
+        <div class="starup-block flex-box">
+          <img class="flex-box-img icons" src="./images/game2.png" alt="" />
+          <h2 class="font-lato">Card Games</h2>
+          <p class="font-lato">
+            It is a rummy style of game in which the aim is to make melds of seven cards of the same value.
+          </p>
+        </div>
+
+        <div class="growth-hacks flex-box">
+          <img class="flex-box-img icons" src="./images/game3.png" alt="" />
+          <h2 class="font-lato">Baloon Stomp</h2>
+          <p class="font-lato">
+            Balloon Stomp is a frenetic game in which players pop the balloons tied around each other's ankles.
+          </p>
+        </div>
+
+        <div class="why Raise Fund flex-box">
+          <img class="flex-box-img icons" src="./images/game4.png" alt="" />
+          <h2 class="font-lato">Rope Pulling</h2>
+          <p class="font-lato">
+            Pulling the rope along with the members of your team against the opposition team each to their side.
+          </p>
+        </div>
+
+        <div class="forum flex-box">
+          <img class="flex-box-img icons" src="./images/game5.png" alt="" />
+          <h2 class="font-lato">Horse Riding</h2>
+          <p class="font-lato">
+            Horse riding or horseback riding, includes the disciplines of riding, driving, and vaulting.
+          </p>
+        </div>
+      </div>
+      <div class="button-container">
+        <button type="submit" class="program-button font-lato">
+          Join CCS Summer Concert
+        </button>
+      </div>
+    </div>
+  </section>
+
+  <div class="section-speakers-heading">
+    <h2 class="font-lato">Featured Speakers</h2>
+    <hr />
+  </div>
```

Comment:
- [ ] The **spacings** of contents within your _Feature Speakers_ section are too wide apart. Kindly adjust the spacing of contents in this section proportionally, as can be seen in the screenshot of the template design below:

**Your Featured Speakers section:**
![image](https://user-images.githubusercontent.com/12745474/132957460-03af1d7a-18f2-42a6-b496-cec88e77ebfe.png)
<br>

**The original Featured Speakers section:**
![image](https://user-images.githubusercontent.com/12745474/132957499-8761be53-601f-4ed3-9274-c30327fac25b.png)